Share/OpenAIapi/openAIapiStruct.swift

```diff
@@ -101,7 +101,7 @@ struct ErrorRootResponse: Decodable{
     let error: ErrorResponse
 }
 
-struct ErrorResponse: Codable{
+struct ErrorResponse: Decodable{
```

Comment:
In Swift, Codable is a type alias for the combination of the Encodable and Decodable protocols. This means that if a Swift struct conforms to Codable, it can be both encoded and decoded to/from an external representation like JSON, plist, or other data formats.